instrumentation-api/src/test/java/io/opentelemetry/instrumentation/api/instrumenter/InstrumenterTest.java

```diff
@@ -286,7 +288,7 @@ void client() {
             .addAttributesExtractor(new AttributesExtractor1())
             .addAttributesExtractor(new AttributesExtractor2())
             .addSpanLinksExtractor(new LinksExtractor())
-            .buildClientInstrumenter(Map::put);
+            .buildClientInstrumenter((map, key, value) -> requireNonNull(map).put(key, value));
```

Comment:
maybe we should try silencing it for tests

Reply:
good idea - done